operations/bosh-lite.yml

```diff
@@ -166,39 +166,14 @@
   - public_networks
   - dns
   - load_balancer
-- type: replace
```

Comment:
This was the only ops-file that we use that broke due to this change, but we did not test all of them.